persistence-modules/spring-data-jpa-query-3/pom.xml

```diff
@@ -8,20 +8,21 @@
     <properties>
```

Comment:
build and properties should be below parent and dependencies. Check jira description.

Reply:
@Bipinkumar27 please check this